Dockerfile

```diff
@@ -31,7 +31,11 @@ COPY stored_requests/data stored_requests/data
 RUN apt-get update && \
     apt-get install -y ca-certificates mtr && \
     apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
-EXPOSE 8000
-EXPOSE 6060
-ENTRYPOINT ["/usr/local/bin/prebid-server"]
-CMD ["-v", "1", "-logtostderr"]
+
```

Comment:
Please do not commit changes to the Dockerfile

Reply:
Sure.  sorry about that